apidef/api_definitions.go

```diff
@@ -749,9 +749,10 @@ const (
 )
 
 type GraphQLProxyConfig struct {
-	AuthHeaders      map[string]string `bson:"auth_headers" json:"auth_headers"`
-	SubscriptionType SubscriptionType  `bson:"subscription_type" json:"subscription_type,omitempty"`
-	RequestHeaders   map[string]string `bson:"request_headers" json:"request_headers"`
+	AuthHeaders              map[string]string `bson:"auth_headers" json:"auth_headers"`
+	SubscriptionType         SubscriptionType  `bson:"subscription_type" json:"subscription_type,omitempty"`
+	RequestHeaders           map[string]string `bson:"request_headers" json:"request_headers"`
+	ReturnErrorsFromUpstream bool              `bson:"return_errors_from_upstream" json:"return_errors_from_upstream"`
```

Comment:
To make this feature proof maybe we can create a new struct here:
```go
UseResponseExtensions GraphQLResponseExtensions `bson:"use_response_extensions" json:"use_response_extensions"`

type GraphQLResponseExtensions struct {
    OnErrorForwarding bool `bson:"on_error_forwarding" json:"on_error_forwarding"`
}
```

which results in
```json
"proxy": {
        "auth_headers": {},
        "request_headers": null,
        "use_response_extensions": {
             "on_error_forwarding": true
        }
    },
```
Feel free to adjust this idea